parser/src/main/java/main/ParserMain.java

```diff
@@ -17,20 +16,19 @@
 
 import java.util.Set;
 
+@Log4j2
 public class ParserMain {
 
-    private static Logger logger = LogManager.getRootLogger();
-
     public static void main(String[] args) {
-        logger.info("Parser application starts");
+        log.info("Parser application starts");
 
         if(args.length == 0) {
-            logger.error("No library name provided!");
+            log.error("No library name provided!");
             System.exit(-1);
```

Comment:
System exit  ? :octopus: is there any reason to close the JVM in this situation ?